### DISCUSSION TITLE
Purge devices - remove devices that havent connected to the server in X amount of time.

### DISCUSSION THREAD

**Discussion by rodrigolzd** [Ylianst/MeshCentral#5538] [Q&A]
I'm trying to remember if there was a general setting for the whole system to purge devices that havent having connected after X amount of time.  

I've searched for "Purge" on both the docs and the issues but didnt thing anything. I honestly think I've seen and used this before. And I'm not talking about the per group funtion. I honestly thought there was a function on the server config for this. I'm walking in circles now.

Any suggestions, maybe the word for it is not "Purge" ?

**Comment by si458** [comment]
you can set it per group, by clicking the group then clicked the pencil next to Features
OR/AND
you can set `autoRemoveInactiveDevices` inside `domain` in your `config.json` but this applies to ALL GROUPS!
![image](https://github.com/Ylianst/MeshCentral/assets/765314/72da7f16-4a26-4332-a4b1-106169b62997)

**Reply by rodrigolzd** [reply]
I think autoRemoveInactiveDevices is what I was looking for and yes, I want this to apply to all groups.

Later on I'll try to add this to the documentation.
